CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+<!--
+SPDX-FileCopyrightText: 2019 The curiu authors
+
+SPDX-License-Identifier: LicenseRef-ISC-curiu
+-->
+
+# Contributing
+
+- Add yourself to the [AUTHORS.md](AUTHORS.md) file.
+- Autoformat new/changed code with [`black`](https://github.com/psf/black).
+- Add SPDX copyright/license headers or `.license` files and check them via [`reuse lint`](https://github.com/fsfe/reuse-tool/tree/v0.5.0#usage).
```

Comment:
Running `black` and `reuse lint` can/should be automated at some point, of course.

Reply:
I can add a ``pre-commit`` hook in a second step which will run black. 
It is also possible to create a bot that will check for it and it will lint the code if we want to format the PR using black if someone is not using the pre-commit hook. And of course, we can enforce that running a test on CI to see if the whole codebase is using black
;)

Reply:
Yes, hanks!
I wanted to look into https://github.com/pre-commit/pre-commit for a long time, but haven't done so yet. Would make it easier than writing/maintaining that stuff ourselves ;).